examples/decoding/plot_haxby_different_estimators.py

```diff
@@ -10,7 +10,13 @@
 
 ### Fetch data using nilearn dataset fetcher ################################
 from nilearn import datasets
-haxby_dataset = datasets.fetch_haxby(n_subjects=1)
+haxby_dataset = datasets.fetch_haxby(n_subjects=2)
```

Comment:
Any good reason why this was changed to 2 ?

I guess you want to be consistent and use something like "First subject functional ..." in your print statement below and all the other haxby examples.